src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,39 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+	public int getTotal() {
+		return total;
+	}
+    private static final int MAX_TAB_SIZE = 12;
+    private static final int EMPTY = -1;
+
+    private final int[] numbers = new int[MAX_TAB_SIZE];
```

Comment:
Original project formatting convention used tabs (should not be changed to spaces). It makes tracking actual changes more difficult.